src/Cache/_ScsDataClient.php

```diff
@@ -565,18 +565,18 @@ public function dictionaryGetFields(string $cacheName, string $dictionaryName, a
             validateCacheName($cacheName);
             validateDictionaryName($dictionaryName);
             validateItems($fields);
-            $dictionaryGetBatchRequest = new _DictionaryGetRequest();
-            $dictionaryGetBatchRequest->setDictionaryName($dictionaryName);
-            $dictionaryGetBatchRequest->setFields($fields);
-            $call = $this->grpcManager->client->DictionaryGet($dictionaryGetBatchRequest, ["cache" => [$cacheName]], ["timeout" => $this->timeout]);
-            $dictionaryGetBatchResponse = $this->processCall($call);
+            $dictionaryGetFieldsRequest = new _DictionaryGetRequest();
+            $dictionaryGetFieldsRequest->setDictionaryName($dictionaryName);
+            $dictionaryGetFieldsRequest->setFields($fields);
+            $call = $this->grpcManager->client->DictionaryGet($dictionaryGetFieldsRequest, ["cache" => [$cacheName]], ["timeout" => $this->timeout]);
+            $dictionaryGetFieldsResponse = $this->processCall($call);
         } catch (SdkError $e) {
             return new CacheDictionaryGetFieldsResponseError($e);
         } catch (Exception $e) {
             return new CacheDictionaryGetFieldsResponseError(new UnknownError($e->getMessage()));
         }
-        if ($dictionaryGetBatchResponse->hasFound()) {
-            return new CacheDictionaryGetFieldsResponseSuccess($dictionaryGetBatchResponse);
+        if ($dictionaryGetFieldsResponse->hasFound()) {
+            return new CacheDictionaryGetFieldsResponseSuccess($dictionaryGetFieldsResponse, fields: $fields);
         }
         return new CacheDictionaryGetFieldsResponseSuccess(null, count($fields));
```

Comment:
This should be returning `CacheDictionaryGetResponseHit` and `CacheDictionaryGetResponseMiss` instead of trying to use the `Success` class to represent two different states.